app/ante/eth.go

```diff
@@ -460,17 +458,21 @@ func (vbd EthValidateBasicDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simu
 		}
 
 		// return error if contract creation or call are disabled through governance
-		if !params.EnableCreate && txData.GetTo() == nil {
+		enableCreate := vbd.evmKeeper.GetEnableCreate(ctx)
```

Comment:
By moving this in here you are calling params for every message. Was there any reason why you decided to move it?